calico_node/tests/st/policy/test_profile.py

```diff
@@ -79,7 +80,8 @@ def tearDown(self):
 
             super(MultiHostMainline, self).tearDown()
 
-    def _test_tags(self):
+    @skip("Tags are deprecated")
```

Comment:
More than deprecated; they've been removed now.

Think this should say something like "TODO: Review if there is a corresponding useful test now that tags have been removed."